tests/e2e/storage.e2e-spec.ts

```diff
@@ -0,0 +1,205 @@
+require('dotenv').config();
+import request from 'supertest';
+import { HttpStatus } from '@nestjs/common';
```

Comment:
Since we are only using @nestjs/common which comes with rxjs for the statuscode we could do an object here like:

`const HttpStatus = {
   OK: 200
}`

Instead of importing the whole package along with rxjs, however, since we are eventually going to join this with WIP (which already has nest), I think it makes sense to have it anyway.

Reply:
TLDR: ignore this comment 😄